spread-tests/integration/task_autostart/task.yaml

```diff
@@ -0,0 +1,11 @@
+
+summary: Dummy service autostart test
+kill-timeout: 2m
+
+execute: |
+    ${SUITE_DIR}/pebble_wrapper.sh logs | grep -e "dummy service log"
```

Comment:
snapd has a script or binary that takes piped output and can match on regex and substrings. We could consider that for our final version. I know this is mostly for a proof of concept.